react/components/Footer/components/FooterVtexLogo.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+class FooterVtexLogo extends React.Component{
```

Comment:
It's better to export the component here as ```export default classFooterVtexLogo...```.

Reply:
Done